cannon/mipsevm/tests/open_mips_tests/test/clone.asm

```diff
@@ -1,26 +0,0 @@
-.section .test, "x"
```

Comment:
We should keep this in order to not lose code coverage for singlethreaded Cannon `clone` functionality. These tests are incredibly useful for debugging VM `step`.

MT-Cannon tests can ignore it for the time being. We can add MT specific MIPS test vectors later.